api/config/server.ts

```diff
@@ -0,0 +1,7 @@
+export default ({ env }) => ({
+  host: env('HOST', '0.0.0.0'),
+  port: env.int('PORT', 1337),
+  app: {
```

Comment:
Can you check the purpose of this and remove if not needed?

Reply:
this is mandatory to keep as taking reference from strapi server confiuration https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations/required/server.html#available-options
this is use by the session middleware for the Users & Permissions plugin and the Documentation plugin.